test/helper/summary.test.ts

```diff
@@ -35,7 +35,7 @@ describe('createErrorSummary', () => {
   });
 
   test('Should return summary of one error and two warnings', () => {
-    ticsConfig.logLevel = 'debug';
+    githubConfig.debugger = 'debug';
```

Comment:
You test against 'debug', but the environment variable woll specify 1. It would be good to check against
undefined, '0' (IsNullOrEmpty()).

Reply:
It is now a boolean, so now I'm setting it to true or false in tests.